### PR TITLE
Added data source for thermodynamic info in tests

### PR DESCRIPTION
**Pull request by stelmo**

### CODE REVIEW

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 10e66875 is formatted properly.

File: test/data_static.jl [hunk 1]
```diff
@@ -74,6 +74,7 @@ test_toyModel() = CoreModel(
 )
 
 const reaction_standard_gibbs_free_energies = Dict(
+    # data from Equilibrator using the E. coli core model's reactions
```
Comment (laurentheirendt) [inline test/data_static.jl:77]: is there a proper source? did you generate the data? How did you generate the data?

**Review by laurentheirendt — APPROVED** [review]

File: test/data_static.jl (hunk 1 above)
Comment (stelmo) [reply test/data_static.jl:77]: no proper source, but I will be a little more thorough

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 35422a6b is formatted properly.

**Review by laurentheirendt — APPROVED** [review]

**Review by exaexa — APPROVED** [review]

**Comment by exaexa** [comment]
this is the way.

**Pull request merged**